sfy-buoy/sfy-artemis/src/main.rs

```diff
@@ -303,7 +303,7 @@ fn main() -> ! {
 
         // XXX: This needs to be adapted to frequency, and queue length. Maybe just remove when we
         // have the remaining space check? Check after Hjeltefjorden deployment.
-        const LOOP_DELAY: u32 = 14 * 40_000;
+        const LOOP_DELAY: u32 = 10 * 40_000;                     ///////////////////////
```

Comment:
The queue length is a little more than 14. While the time it takes to fill up a 1024 elements long buffer with 26 Hz is about 40 seconds (40_000 ms). This is the output frequency from the FIR filter, which is still 26 Hz (or 52 Hz if so configured) even if you run the IMU faster. So this should stay the same.